avro-fastserde/src/test/java/com/linkedin/avro/fastserde/FastSerdeCacheTest.java

```diff
@@ -68,4 +70,19 @@ public void testBuildFastSpecificDeserializerWithCorrectClasspath() throws Excep
     FastSerdeCache cache = FastSerdeCache.getDefaultInstance();
     cache.buildFastSpecificDeserializer(TestRecord.SCHEMA$, TestRecord.SCHEMA$);
   }
+
+  @Test
+  public void testSchemaNumLimit() throws Exception {
+    FastSerdeCache cache = new FastSerdeCache(100);
+    Assert.assertEquals(cache.getGeneratedFastClassNumLimit(), 100);
+
+    Supplier<String> supplier = new Supplier<String>() {
+      @Override
+      public String get() {
+        return ".";
+      }
+    };
+    FastSerdeCache cache2 = new FastSerdeCache(Runnable::run, supplier, 100);
+    Assert.assertEquals(cache2.getGeneratedFastClassNumLimit(), 100);
+  }
```

Comment:
I don't think we need this test. If you have logic in your constructor, you should test it. Simply setting private properties is not _logic_.

Reply:
If you really want to test the logic, you can set the limit to be 1 and try to generate two fastdeserializers. Then you can check `buildGenericSerializer` would not be called for the second schema.

Reply:
I'll just remove it